README.md

```diff
@@ -111,5 +111,35 @@ icon: https://pbs.twimg.com/profile_images/929088242456190976/xjkS2L-0_400x400.j
 url: https://github.com/ws-skeleton/che-editor-theia/releases/download/untagged-892e01b21d0145207b0f/che-editor-plugin.tar.gz
 ```
 
+### Plugin meta YAML structure:
+Here is an overview of all fields that can be present in plugin meta YAML files
+
+|field name in YAML | description | mandatory |
+|-------|-------|-------|
+| id | plugin ID | yes |
+| version | plugin version | yes |
+| type | plugin type | yes|
+| name | plugin name | yes|
+| title| plugin title | yes |
+| description| plugin description | yes |
+| icon| URL to plugin icon | yes |
```

Comment:
I think it makes sense to add info that is MUST be svg

Reply:
fixed